files/project_team/IFCTunnelDep3_Geo_BookA.ifc

```diff
@@ -0,0 +1,663 @@
+#351= IFCCARTESIANPOINT((0.0,0.0,0.0));
+#352= IFCPRODUCTDEFINITIONSHAPE($,$,(#353));
+#353= IFCSHAPEREPRESENTATION(#205,'Axis','Segment',(#65));
+#354= IFCALIGNMENTVERTICALSEGMENT($,$,27611.495733,127.483423,90.720889,0.0114998772631749,0.00400012404346658,-17000.0,.CIRCULARARC.);
+#355= IFCALIGNMENTSEGMENT('3GSxdC0TX3Ef5I36irRSXF',$,'E',$,$,#356,#359,#361);
+#356= IFCLOCALPLACEMENT($,#357);
+#357= IFCAXIS2PLACEMENT3D(#358,$,$);
+#358= IFCCARTESIANPOINT((0.0,0.0,0.0));
+#359= IFCPRODUCTDEFINITIONSHAPE($,$,(#360));
+#360= IFCSHAPEREPRESENTATION(#205,'Axis','Segment',(#73));
+#361= IFCALIGNMENTVERTICALSEGMENT($,$,27738.979156,91.146129,91.708871,0.00400000265238435,0.00400000265238435,$,.CONSTANTGRADIENT.);
+#362= IFCALIGNMENTSEGMENT('19DYyqJRr0XQ58KxO0MwW2',$,'E',$,$,#363,#366,#368);
+#363= IFCLOCALPLACEMENT($,#364);
+#364= IFCAXIS2PLACEMENT3D(#365,$,$);
+#365= IFCCARTESIANPOINT((0.0,0.0,0.0));
+#366= IFCPRODUCTDEFINITIONSHAPE($,$,(#367));
+#367= IFCSHAPEREPRESENTATION(#205,'Axis','Segment',(#81));
+#368= IFCALIGNMENTVERTICALSEGMENT($,$,27830.125285,115.497574,92.073456,0.003999974433555,-0.00649997430106127,-11000.0,.CIRCULARARC.);
+#369= IFCALIGNMENTSEGMENT('0rQCRcDMnFC9YCIxWfVyHM',$,'E',$,$,#370,#373,#375);
+#370= IFCLOCALPLACEMENT($,#371);
+#371= IFCAXIS2PLACEMENT3D(#372,$,$);
+#372= IFCCARTESIANPOINT((0.0,0.0,0.0));
+#373= IFCPRODUCTDEFINITIONSHAPE($,$,(#374));
+#374= IFCSHAPEREPRESENTATION(#205,'Axis','Segment',(#89));
+#375= IFCALIGNMENTVERTICALSEGMENT($,$,27945.622859,68.745554,91.929088,-0.00650000252063129,-0.00650000252063129,$,.CONSTANTGRADIENT.);
+#376= IFCALIGNMENTSEGMENT('2JxTLaMdb7cRXgBOcAeEGa',$,'E',$,$,#377,#380,#382);
+#377= IFCLOCALPLACEMENT($,#378);
+#378= IFCAXIS2PLACEMENT3D(#379,$,$);
+#379= IFCCARTESIANPOINT((0.0,0.0,0.0));
+#380= IFCPRODUCTDEFINITIONSHAPE($,$,(#381));
+#381= IFCSHAPEREPRESENTATION(#205,'Axis','Segment',(#97));
+#382= IFCALIGNMENTVERTICALSEGMENT($,$,28014.368413,64.39231,91.482242,-0.00650003642158459,-0.00879996519264084,-28000.0,.CIRCULARARC.);
+#383= IFCALIGNMENTSEGMENT('313UCQdYT0xAVxKqPCOXWe',$,'E',$,$,#384,#387,#389);
+#384= IFCLOCALPLACEMENT($,#385);
+#385= IFCAXIS2PLACEMENT3D(#386,$,$);
+#386= IFCCARTESIANPOINT((0.0,0.0,0.0));
+#387= IFCPRODUCTDEFINITIONSHAPE($,$,(#388));
+#388= IFCSHAPEREPRESENTATION(#205,'Axis','Segment',(#105));
+#389= IFCALIGNMENTVERTICALSEGMENT($,$,28078.760723,444.026785,90.989641,-0.00879999909478706,-0.00879999909478706,$,.CONSTANTGRADIENT.);
+#390= IFCALIGNMENTSEGMENT('3OQZ4r7BPCsOW_vuQRgEEL',$,'E',$,$,#391,#394,#396);
+#391= IFCLOCALPLACEMENT($,#392);
+#392= IFCAXIS2PLACEMENT3D(#393,$,$);
+#393= IFCCARTESIANPOINT((0.0,0.0,0.0));
+#394= IFCPRODUCTDEFINITIONSHAPE($,$,(#395));
+#395= IFCSHAPEREPRESENTATION(#205,'Axis','Segment',(#113));
+#396= IFCALIGNMENTVERTICALSEGMENT($,$,28522.787508,25.004436,87.082206,-0.00879998184706311,-0.00779970093772224,25000.0,.CIRCULARARC.);
+#397= IFCALIGNMENTSEGMENT('2Egl8W$NfFiuvG5vzE4piJ',$,'E',$,$,#398,#401,#403);
```

Comment:
There are plenty of `IfcAlignmentSegment`s with the same name `E`. I don't think that it is against the schema definitions, but I would welcome unique names.

Reply:
Agree, but again, alignment data comes from the LandXML=>IFC conversion by Peter Bonsma. I just clone the stuff in.